lib/puma/const.rb

```diff
@@ -112,28 +111,28 @@ module Const
     WRITE_TIMEOUT = 10
 
     # The original URI requested by the client.
-    REQUEST_URI= 'REQUEST_URI'.freeze
-    REQUEST_PATH = 'REQUEST_PATH'.freeze
-    QUERY_STRING = 'QUERY_STRING'.freeze
-    CONTENT_LENGTH = "CONTENT_LENGTH".freeze
+    REQUEST_URI= 'REQUEST_URI'
+    REQUEST_PATH = 'REQUEST_PATH'
+    QUERY_STRING = 'QUERY_STRING'
```

Comment:
Suggestion: now that we are touching these lines, let's take the opportunity to make things a tiny bit more consistent by using `"` over `'`

```suggestion
    REQUEST_URI= "REQUEST_URI"
    REQUEST_PATH = "REQUEST_PATH"
    QUERY_STRING = "QUERY_STRING"
```